hydra_plugins/hydra_wandb_sweeper/config.py

```diff
@@ -1,30 +1,95 @@
-import dataclasses
-import typing
+from dataclasses import dataclass, field
+from typing import Any, Dict, MutableSequence, Optional
 
-import omegaconf
-from hydra.core import config_store
+from hydra.core.config_store import ConfigStore
+from omegaconf import DictConfig, ListConfig
 
 
-@dataclasses.dataclass
+@dataclass
+class WandbParameterSpec:
+    """Representation of all the options to define
+    a Wandb parameter.
+    """
+
+    # (int or float) Maximum and minimum values.
+    # if int, for int uniform-distributed hyperparameters. If float, for uniform-distributed hyperparameters.
+    min: Optional[float] = None
+    max: Optional[float] = None
+
+    # (float) Mean parameter for normal or log normal-distributed hyperparameters.
+    mu: Optional[float] = 0
+
+    # (float) Standard deviation parameter for normal or log normal-distributed hyperparameters.
+    sigma: Optional[float] = 1
+
+    # (float) Quantization step size for quantized hyperparameters.
+    q: Optional[float] = 1
+
+    # Specify how values will be distributed
+    # Supported distributions and required parameters
+    # "constant": ["value"],
+    # "categorical": ["values"],
+    # "int_uniform": ["min", "max"],
+    # "uniform": ["min", "max"],
+    # "q_uniform": ["min", "max", "q"],
+    # "log_uniform": ["min", "max"],
+    # "q_log_uniform": ["min", "max", "q"],
+    # "inv_log_uniform": ["min", "max"],
+    # "normal": ["mu", "sigma"],
+    # "q_normal": ["mu", "sigma", "q"],
+    # "log_normal": ["mu", "sigma"],
+    # "q_log_normal": ["mu", "sigma", "q"],
```

Comment:
If this section of code is not needed, please remove it.

Reply:
It's not code, it's just comments to inform the support distributions and their associated required parameters. It's meant for those who decide to look into this code in the future.